module/actor/sheets/WitcherActorSheet.js

```diff
@@ -347,7 +347,7 @@ export default class WitcherActorSheet extends ActorSheet {
 
             if (verbalCombat.skill) {
               rollFormula += !displayRollDetails ? ` +${vcStat} +${vcSkill}` : ` +${vcStat}[${game.i18n.localize(vcStatName)}] +${vcSkill}[${game.i18n.localize(vcSkillName)}]`
-              rollFormula = addAllModifiers(this.actor, verbalCombat.skill.name, rollFormula)
+              rollFormula = this.actor.addAllModifiers(verbalCombat.skill.name, rollFormula)
```

Comment:
In this step the rollFormula get enhanced with additional data. Dont know why I added it as a parameter since it is not needed. I will fix that so that it will only be

``rollFormula += addAllModifiers(skillname)``